code/controllers/CronTaskController.php

```diff
@@ -9,21 +10,11 @@ class CronTaskController extends Controller
 {
 
     /**
-     * If this controller is in quiet mode
+     * Setting this to true will make the controller print debug information
      *
      * @var bool
      */
-    protected $quiet = false;
```

Comment:
Still changes the behaviour, so it's a major change IMO, and it's against a 2.0 branch.